static/app/views/replays/details.tsx

```diff
@@ -70,8 +70,20 @@ function ReplayDetails() {
   );
 }
 
+const handleLayout = (): LayoutModes => {
+  const {
+    screen: {height, width},
```

Comment:
lets not base our recommendation on the _screen_ size, instead the browser window size would be interesting.

Not too many screens on laptops or on people's desks are more tall than wide. I guess something like an iPad could be flipped either direction, but that's not our userbase.